app/controllers/validations.js

```diff
@@ -0,0 +1,16 @@
+exports.validateUser = object => {
+  const errorMsgs = [],
+    validMail = /([A-Za-z][0-9])|([0-9][A-Za-z])@wolox.com.ar\s*$/,
+    validPass = /([A-Za-z]+[0-9]+)|([0-9]+[A-Za-z]+)/;
+  Object.keys(object).forEach(key => {
```

Comment:
Maybe a function for do this (you have to do this kind of things multiple times)